proto/minder/v1/minder.proto

```diff
@@ -2096,6 +2106,9 @@ enum ProviderType {
     PROVIDER_TYPE_GIT = 3 [(name) = "git"];
     PROVIDER_TYPE_OCI = 4 [(name) = "oci"];
     PROVIDER_TYPE_REPO_LISTER = 5 [(name) = "repo-lister"];
+    PROVIDER_TYPE_IMAGE_LISTER = 6 [(name) = "image-lister"];
+    PROVIDER_TYPE_GHCR = 7 [(name) = "ghcr"];
```

Comment:
GHCR and Dockerhub are Provider classes, not types/traits, right?

Reply:
Oops, leftovers